src/favoriteManager.js

```diff
@@ -15,3 +16,29 @@ export function toggle(mentor, favs) {
 export function get() {
   return getFavorites();
 }
+
+export async function readAndUpdateFavMentorsFromLocalStorage(){
+  let isMentorListUpdated = false;
+  const favsFromLocal = window.localStorage.getItem(LOCAL_FAV_KEY);
+  if (favsFromLocal) {
+    isMentorListUpdated = true;
+    const favMentorsFromLocalStorage = JSON.parse(favsFromLocal);
+    window.localStorage.removeItem(LOCAL_FAV_KEY);
+
+    let timeOut = 0;
+    let updateFavoritesPromises = [];
+
+    favMentorsFromLocalStorage.forEach((mentorId) => {
+      const updateFavoritePromise = new Promise((resolve, reject) => {
+          setTimeout(async (mentorId)=>{
+            const res = await addMentorToFavorites(mentorId);
+            resolve(res);
+          }, timeOut, mentorId);
+      })
+      updateFavoritesPromises.push(updateFavoritePromise);
+      timeOut += 300;
+    });
+    await Promise.all(updateFavoritesPromises).then((values) => console.log(values));
```

Comment:
Do we need to wait for it? We already have the favorites list for the ui. The call for the server to update tue list can be in "background".

Reply:
Yes, this makes sense. I will make the changes. Thanks!

Reply:
Done! @moshfeu please review.